src/docs/reference/core/Placement/README.md

```diff
@@ -233,6 +233,22 @@ Result:
 3 days ago
 ```
 
+### Duration shapes
```

Comment:
This seems out of place, but it means the paragraph above too. This file is for Placement.

Reply:
I placed it along with the related shapes docs `DateTime` and `TimeSpan` shapes.

Reply:
I know, both should be moved, or at least don't put it there too.

Reply:
Any suggestion on where should be move all three to?

Reply:
I create a Display management file and moved the Shapes to it